Projects/UOContent/Skills/SkillCheck.cs

```diff
@@ -299,17 +347,17 @@ public static void IncreaseStat(Mobile from, Stat stat, bool atrophy)
                     {
                         if (CanLower(from, Stat.Dex) && (from.RawDex < from.RawInt || !CanLower(from, Stat.Int)))
                         {
-                            --from.RawDex;
+                            from.RawDex = DecreaseStatOrZero(from.RawDex);
```

Comment:
The stat property itself already has these checks. So we don't need to do them here.